turbinia/evidence.py

```diff
@@ -282,12 +304,31 @@ def __init__(self, project=None, zone=None, disk_name=None, *args, **kwargs):
     super(GoogleCloudDisk, self).__init__(*args, **kwargs)
     self.cloud_only = True
 
+    self._attached_path = None
+
   def _preprocess(self):
-    self.local_path = google_cloud.PreprocessAttachDisk(self.disk_name)
+    self._attached_path = google_cloud.PreprocessAttachDisk(self.disk_name)
```

Comment:
What do you think about using just `device_path` instead of `_attached_path`?

Reply:
Sure. I'm keeping the underscore in front of attributes names that we keep around just so we can later use them when doing postprocess